crates/stackable-telemetry/src/tracing/mod.rs

```diff
@@ -446,6 +469,26 @@ impl TracingBuilder<builder_state::Config> {
             console_log_settings: console_log_settings.into(),
             otlp_log_settings: self.otlp_log_settings,
             otlp_trace_settings: self.otlp_trace_settings,
+            file_log_settings: self.file_log_settings,
+            _marker: self._marker,
+        }
+    }
+
+    /// Enable the file output tracing subscriber and set the default
+    /// [`LevelFilter`][1] which is overridable through the given environment
+    /// variable.
+    ///
+    /// [1]: tracing_subscriber::filter::LevelFilter
+    pub fn with_file_output(
+        self,
+        file_log_settings: FileLogSettings,
+    ) -> TracingBuilder<builder_state::Config> {
+        TracingBuilder {
+            service_name: self.service_name,
+            console_log_settings: self.console_log_settings,
+            file_log_settings,
+            otlp_log_settings: self.otlp_log_settings,
+            otlp_trace_settings: self.otlp_trace_settings,
```

Comment:
note: #940 changed the API interface of these functions and as such I suggest to also change it here to be in line.

```suggestion
    pub fn with_file_output(
        self,
        file_log_settings: impl Into<FileLogSettings>,
    ) -> TracingBuilder<builder_state::Config> {
        TracingBuilder {
            service_name: self.service_name,
            console_log_settings: self.console_log_settings,
            file_log_settings: file_log_settings.into(),
            otlp_log_settings: self.otlp_log_settings,
            otlp_trace_settings: self.otlp_trace_settings,
```